lib/factory_bot_profiler/reporters/simple_reporter.rb

```diff
@@ -17,18 +18,30 @@ def report
         @collector.highest_total_time(N).each do |stat|
           puts "    - :#{stat.name} factory took #{(stat.total_time).round(2)} seconds overall"
           puts "       (#{stat.total_child_time.round(2)} seconds spent in associated child factories)" unless stat.total_child_time.zero?
+          puts
         end
         puts
         puts "  Slowest factories on average:"
         @collector.highest_average_time(N).each do |stat|
           puts "    - :#{stat.name} factory took #{(stat.average_time).round(2)} seconds on average"
+          stat.individual_child_times.each do |factory_name, time|
+            puts "      - #{time.round(2)} spent in :#{factory_name} called #{stat.individual_child_count[factory_name]}/#{stat.count} time(s)"
+          end
+          puts
         end
         puts
         puts "  Most used factories:"
         @collector.highest_count(N).each do |stat|
           puts "    - :#{stat.name} factory ran #{stat.count} times"
+          puts
         end
       end
+
+      private
+
+      def puts(...)
```

Comment:
Passing in io makes sense to me (I was starting to head in the same direction over in https://github.com/composerinteralia/factory_bot_profiler/compare/merging?expand=1) but overriding `puts` seems overly sneaky to me.